app/Importer/UserImporter.php

```diff
@@ -38,7 +38,7 @@ public function createUserIfNotExists(array $row)
         $this->item['email'] = $this->findCsvMatch($row, 'email');
         $this->item['phone'] = $this->findCsvMatch($row, 'phone_number');
         $this->item['jobtitle'] = $this->findCsvMatch($row, 'jobtitle');
-        $this->item['activated'] = $this->findCsvMatch($row, 'activated');
+        $this->item['activated'] = ($this->findCsvMatch($row, 'activated')) ? 1 : 0;
```

Comment:
I think we want to use the `fetchHumanBoolean()` method here?

Reply:
The fetchHumanBoolean() method didn't existed in the Importer.php class of Develop. I ported it from master and now it's working.